pyodide_build/pypabuild.py

```diff
@@ -242,6 +242,23 @@ def make_command_wrapper_symlinks(symlink_dir: Path) -> dict[str, str]:
     return env
 
 
+@contextmanager
+def _get_symlink_dir(env: dict[str, str], build_dir: Path | None):
+    if build_dir:
+        # If we're running under build-recipes, leave the symlinks in
+        # the build directory. This helps with reproducing.
+        symlink_dir = build_dir / "symlinks"
```

Comment:
Maybe let's make the name more explicit and unique, such as `.pyodide_symlinks`.

Reply:
Well we delete and recreate this build directory, the only things ever in it are:
1. the source folder
2. the source archive
3. now this symlink folder

I don't really see any reason to put a `.` in the name. But maybe I'll call it `pywasmcross_symlinks`.